### PR TITLE
Adding recipe for seismon

### PR DESCRIPTION
**Pull request by mcoughlin**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/seismon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - utils
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/gwdetchar/seismon/
+  summary: A python package for mitigating the effects of earthquakes on GW detectors
+  license: GPL-2.0-or-later
```
Comment (duncanmmacleod) [inline recipes/seismon/meta.yaml:49]: This needs to be changed [[ref](https://github.com/gwdetchar/seismon/blob/v0.0.3/LICENSE)]:

```suggestion
  license: GPL-3.0-or-later
```

File: recipes/seismon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: ceff57b53e3463a84408425f0ad3feba2c5eb181bef3176af41e5aca15274a23
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (duncanmmacleod) [inline recipes/seismon/meta.yaml:19]: This project needs `setuptools` to build

```suggestion
    - pip
    - python
    - setuptools
```

File: recipes/seismon/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - passlib
+    - python
+    - redis-py
+    - scipy >=0.12.1
+    - simplejson
+
+test:
+  imports:
+    - seismon
+    - utils
```
Comment (duncanmmacleod) [inline recipes/seismon/meta.yaml:41]: This should be removed, anticipating a future release that includes https://github.com/gwdetchar/seismon/pull/5.

```suggestion
```

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks.

**Comment by ocefpaf** [comment]
I guess this cannot be noarch b/c of the script, right?

**Pull request merged**